docs/buildah-push.md

```diff
@@ -66,6 +66,10 @@ After copying the image, write the digest of the resulting image to the file.
 
 Don't compress copies of filesystem layers which will be pushed.
 
+**--encryption-key** *key*
+
+A reference prefixed with the encryption protocol to use. The supported protocols are JWE, PGP and PKCS7. For instance, jwe:/path/to/key.pem or pgp:admin@example.com or pkcs7:/path/to/x509-file. This feature is still *experimental*.
```

Comment:
“reference”?

Reply:
It would be nice to have an explanation of JWE, PGP and PKCS7 and/or man page references.  Ditto through out.

Reply:
Added RFC reference numbers for the protocols. Would you like to add the reference to https://github.com/opencontainers/artifacts/pull/15 for context? Or will the RFCs be sufficient for this?

**--encryption-key** *key*

The [protocol:keyfile] to use for image encryption. The protocol can specifies the encryption protocol, which can be  JWE(RFC7516), PGP(RFC4880), and PKCS7(RFC2315). For instance, jwe:/path/to/key.pem or pgp:admin@example.com or pkcs7:/path/to/x509-file.